package.json

```diff
@@ -51,6 +51,7 @@
     "inherits": "2.0.1",
     "jade": "pugjs/jade#29784fd",
     "jed": "1.0.2",
+    "jshashes": "1.0.5",
```

Comment:
Calling shrinkwrap directly can lead to huge diffs, as we can see:
<img width="440" alt="screen shot 2016-02-05 at 4 26 35 pm" src="https://cloud.githubusercontent.com/assets/1270189/12862979/47741240-cc25-11e5-94bd-e5a50d8ea55d.png">

Please checkout npm-shrinkwrap.json again, and then follow the clingwrap instructions. The top set of instructions on the readme are better suited for bumping any ^x.x.x dependencies.

```
#install clingwrap globally
npm install -g clingwrap
npm install jshashes@1.0.5 --save
clingwrap jshashes
```